### PR TITLE
Add VID/PID reqs to README.md

### PR DESCRIPTION
**Pull request by sarah-iot**
Update for https://github.com/google-home/sample-app-for-matter-android/issues/11.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,15 +50,22 @@ To clone the repository, complete the following steps:
...
+    [Verify Matter Modules & Services](https://developers.home.google.com/matter/verify-services)
+    guide.
+*   Build a Matter device with On/Off capabilities. This Codelab was tested with an ESP32 and a
+    virtual device.
+    *   [Building a Matter Virtual Device](https://developers.home.google.com/codelabs/matter-device-virtual)
+    *   [Build an Espressif Device](https://developers.home.google.com/matter/vendors/espressif)
+*   [Create a Matter integration](https://developers.home.google.com/matter/integration/create).
+    For **Configuration**, use `0xFFF1` as your Vendor ID
+    (VID) and `0x8001` as your Product ID (PID). These are the ESP32 defaults
+    for the M5Stack `all-clusters-app`.
```
Comment (pierredelisle) [inline README.md:68]: We should provide more details about the matter-virtual-device we use for testing. 
Which one we build as well as its PID.

**Review by pierredelisle — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sarah-iot) [reply README.md]: Thanks - done.

File: README.md [hunk 2]
```diff
@@ -50,15 +50,25 @@ To clone the repository, complete the following steps:
...
-the [Build an Android App for Matter](https://developers.home.google.com/codelabs/matter-sample-app)
-Codelab.
-
-To make sure that your device has the latest Matter support, review the
-[Verify Matter Modules & Services](https://developers.home.google.com/matter/verify-services) guide.
+*   For an overview of the user interface and features, refer to
+    the [Google Home Sample App for Matter Guide](https://developers.home.google.com/samples/matter-app).
+*   To review code samples and start building, refer to
+    the [Build an Android App for Matter](https://developers.home.google.com/codelabs/matter-sample-app)
+    Codelab.
```
Comment (Vyrastas) [inline README.md:57]: I would actually move these two after the other bullets below.  Reason being it seems users are going straight here and not the guide/codelab.  Let's make the things that are blocking them the first items in the Get Started list, i.e. make sure you have done these things before doing anything else.

File: README.md [hunk 3]
```diff
@@ -50,15 +50,25 @@ To clone the repository, complete the following steps:
...
+*   To make sure that your device has the latest Matter support, review the
+    [Verify Matter Modules & Services](https://developers.home.google.com/matter/verify-services)
+    guide.
+*   [Create a Matter integration](https://developers.home.google.com/matter/integration/create) and
+    build a Matter device with On/Off capabilities. This sample app works with a virtual device
+    and an ESP32.
+    *   [Build a Matter Virtual Device](https://developers.home.google.com/codelabs/matter-device-virtual)
+        with the `rootnode_dimmablelight_bCwGYSDpoe` app. For **Create new project** >
+        **Configuration** in the [Developer Console](https://console.home.google.com/projects),
```
Comment (Vyrastas) [inline README.md:66]: The console guidance of "Create new project" seems out of sync with the "Create a Matter integration" link above, in which you should've already created a project.  Same with the other below.

File: README.md [hunk 4]
```diff
@@ -50,15 +50,25 @@ To clone the repository, complete the following steps:
...
-[Verify Matter Modules & Services](https://developers.home.google.com/matter/verify-services) guide.
+*   For an overview of the user interface and features, refer to
+    the [Google Home Sample App for Matter Guide](https://developers.home.google.com/samples/matter-app).
+*   To review code samples and start building, refer to
+    the [Build an Android App for Matter](https://developers.home.google.com/codelabs/matter-sample-app)
+    Codelab.
+*   To make sure that your device has the latest Matter support, review the
+    [Verify Matter Modules & Services](https://developers.home.google.com/matter/verify-services)
+    guide.
+*   [Create a Matter integration](https://developers.home.google.com/matter/integration/create) and
```
Comment (Vyrastas) [inline README.md:61]: [Create a Matter integration] in the Home Developer Console and build...

**Review by Vyrastas — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (sarah-iot) [reply README.md]: Thanks - pushed an update; LMKWYT.

**Review by mauimauer — APPROVED** [review]

**Review by pierredelisle — APPROVED** [review]

**Pull request merged**